CONTRIBUTING.md

```diff
@@ -247,6 +247,61 @@ The tags used in the commit message shall follow the same scheme as the tags
 present in the `WIP` branch names, described in the
 [Branching Scheme](#branching-guidelines).
 
+### Sign your work
+
+Signing your work involves adding a line at the end of your patch's git commit
```

Comment:
I would simply say:
In order to contribute to the project, you must sign your work.  By signing your work, you certify to the statements set out in the Developer Certificate of Origin below (from [developercertificate.org](http://developercertificate.org/).